src/views/home/components/EditionCard.vue

```diff
@@ -75,6 +89,15 @@ export default class EditionCard extends Vue {
 
     }
 
+    private get user(): boolean {
+        return this.$state.session.user ? true : false;
```

Comment:
`return !!this.$state.session.user` does the same, in a shorter way. It is a Javascript idiom, so you should use it.

Reply:
Although it sounds illogical, only adding this line, with this exact format: :visible="currentEdition !== null" solves the problem of the copy window displayed from 2nd click and on.
Also this format  :visible="currentEdition" results in this exception:
Vue warn]: Invalid prop: type check failed for prop "visible". Expected Boolean, got Object

Reply:
So use !!currentEdition, it turns anything to a boolean based on its truethiness.